src/modules/system/System.cpp

```diff
@@ -39,7 +39,18 @@
 #if defined(LOVE_ANDROID)
 #include "common/android.h"
 #elif defined(LOVE_LINUX)
+
+#ifdef __has_include
```

Comment:
What's the availability of `__has_include` like? Is it possible for `posix_spawnp` to exist but `__has_include` isn't supported?

Reply:
Yes, they're orthogonal: `__has_include` is a C++17 language feature, `posix_spawn` is a function available in some libc implementations.

Reply:
clang and gcc have had `__has_include` for a long time (way before it was standardized in C++17)